features/support/object_model_pages/shop/employee/registration/ee_family_information.rb

```diff
@@ -155,6 +155,10 @@ def self.dependent_select_dc_state
     'div.selectric-open li[data-index="10"]'
   end
 
+  def self.dependent_select_ma_state
+    'div.selectric-open li[data-index="24"]'
```

Comment:
this one might not allow for a data-cuke selector, so we probably can't get that in there

I am wondering about the `24` -- is there a chance this test might fail because the index ends up being 23 or 25?